src/Integration/HostedCommandControllerBase.cs

```diff
@@ -32,10 +32,10 @@ protected HostedCommandControllerBase(IServiceProvider serviceProvider)
                 throw new ArgumentNullException(nameof(serviceProvider));
             }
 
-            this.ServiceProvider = serviceProvider;
+            this.host = serviceProvider;
```

Comment:
Was this deliberate? It's confusing - there is already an _IHost_ (and the property name would normally be capitalised).

Reply:
No! Good catch. Not sure what happened here.